application/src/main/kotlin/com/studentcenter/weave/application/service/application/UserRefreshTokenApplicationService.kt

```diff
@@ -0,0 +1,51 @@
+package com.studentcenter.weave.application.service.application
+
+import com.studentcenter.weave.application.common.exception.AuthExceptionType
+import com.studentcenter.weave.application.port.inbound.UserRefreshTokenUseCase
+import com.studentcenter.weave.application.port.outbound.UserRefreshTokenRepository
+import com.studentcenter.weave.application.service.domain.UserDomainService
+import com.studentcenter.weave.application.service.util.UserTokenService
+import com.studentcenter.weave.domain.entity.User
+import com.studentcenter.weave.support.common.exception.CustomException
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import java.util.*
+
+@Service
+class UserRefreshTokenApplicationService(
+    private val userDomainService: UserDomainService,
+    private val userTokenService: UserTokenService,
+    private val userRefreshTokenRepository: UserRefreshTokenRepository,
+) : UserRefreshTokenUseCase {
+
+    @Transactional
+    override fun invoke(command: UserRefreshTokenUseCase.Command): UserRefreshTokenUseCase.Result {
+        val refreshToken: String = command.refreshToken
+        val refreshTokenUserId: UUID = userTokenService
+            .resolveRefreshToken(refreshToken)
+            .userId
+
+        val s = userRefreshTokenRepository.findByUserId(refreshTokenUserId)
+            validateRefreshTokenExists(s)
```

Comment:
p4) `exist`로 처리하는게 더 적절해보이는데 어떠세요?
```suggestion
        val exist = userRefreshTokenRepository.existByUserId(refreshTokenUserId)
        if (exist.not()) {
       		 // 로직
        }
```